dreamboat/main.py

```diff
@@ -0,0 +1,90 @@
+import pycurl
+import json
+import time
+import logging
+import ConfigParser
+import thread
+
+CONFIG_FILE = 'config'
+
+def make_logger(log_name, log_level):
+    # setup logger
+    logger = logging.getLogger(log_name)
+    logger.setLevel(log_level)
+    # create console handler and set level to debug
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.DEBUG)
+    # create formatter
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    # add formatter to ch
+    ch.setFormatter(formatter)
+    # add ch to logger
+    logger.addHandler(ch)
+    return logger
+
+def give_life(source_url, data, DELAY, log_level):
```

Comment:
Be myyyyyyyyy, frankensteeiinn...  'cause I'm hungry for love and its feedin' ti-i-i-mmee.